docker/hailgenetics/mirror_images.sh

```diff
@@ -42,6 +42,8 @@ images=(
     "python-dill:3.10-slim"
     "hail:${HAIL_PIP_VERSION}"
     "hailtop:${HAIL_PIP_VERSION}"
+    "vep/grch37/85/vep:${HAIL_PIP_VERSION}"
```

Comment:
I might be wrong, but I think the directory structure was mostly so that all our dockerfiles could just be called `Dockerfile`, and that's ultimately separate from the name of the image. I personally think the image should still be named `vep-grch37-85`